### DISCUSSION TITLE
Syntax Page source

### DISCUSSION THREAD

**Discussion by adriansnyman** [Zavy86/WikiDocs#187] [Ideas] [answered]
We are currently using WikiDocs as a kind of a "brain dump" for users in our department.
However, there are many requests for syntax examples - and each time I refer them to the Syntax page.

I was thinking however - could we not put the syntax page directly on our wiki, bypassing any need for internet/external reference ?
If so - is it possible to dump the syntax page source into a folder in the codebase ?

**Comment by Zavy86** [comment, marked as answer]
Hi Adrian, we are working on it.. ;)
I attach the syntax source code for the moment..

[syntax.md](https://github.com/user-attachments/files/16962661/syntax.md)

**Reply by adriansnyman** [reply]
Excellent ! Thanks so much !
